python-challenge-bypass-ristretto.nix

```diff
@@ -1,12 +1,6 @@
 # A basic packaging of this very project: Python bindings to the Rust
 # Ristretto implementation.
 { ristretto, git, python, pythonPackages, setuptools_scm, milksnake, cffi, attrs, testtools, hypothesis }:
-let
-  # Older nixpkgs have Rust crates with everything in the default output.
-  # Newer nixpkgs have Rust crates with a "lib" output and all of the good
-  # stuff is *there*.
-  ristretto-lib = if ristretto ? lib then ristretto.lib else ristretto;
```

Comment:
This was a nice idea but it turned out to be wrong.  We never used newer nixpkgs before so I never noticed that some stuff was missing.